memory/org.eclipse.cdt.debug.ui.memory.search/src/org/eclipse/cdt/debug/ui/memory/search/MemorySearchResultsPage.java

```diff
@@ -178,16 +178,19 @@ public Object[] getElements(Object inputElement) {
 			public void selectionChanged(final SelectionChangedEvent event) {
 				IMemoryRenderingContainer containers[] = ((IMemorySearchQuery) fQuery).getMemoryView()
 						.getMemoryRenderingContainers();
-				if (containers == null || containers.length == 0) return;
+				if (containers == null || containers.length == 0)
```

Comment:
@LizzMre Thanks for applying CDT's coding standards. I am curious as to how you have your editor setup? The CDT projects are configured by default to auto save with this formatting so I was wondering if you are using Eclipse to edit, and if you are if you are purposely bypassing the save actions and formatting rules in the project, or is there something wrong with the CDT developer setup that means the save actions are not working in your use case?

Reply:
@jonahgraham I used Eclipse, but I forgot I had a customized formatting added for the code. I also had a few more projects opened. I don't think it's something wrong with the CDT developer setup.

Reply:
Great. thanks for letting me know!